<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MaterialDialog</class>
 <widget class="QDialog" name="MaterialDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>551</width>
    <height>410</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>551</width>
    <height>410</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>551</width>
    <height>410</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>General Store Item</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#MaterialDialog{
background-color: rgb(47,47,47);
}</string>
  </property>
  <widget class="QPushButton" name="AcceptButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>336</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	color:white;
	border: 2px solid black;
	border-radius: 8px;
	background-color: rgba(220, 0, 0, 180);
}
QPushButton:hover {
	background-color: rgba(180, 0, 0, 200);
}
QPushButton:pressed {
	background-color: rgba(140, 0, 0, 200);
}</string>
   </property>
   <property name="text">
    <string>Accept</string>
   </property>
   <property name="iconSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="LabelStoreItemName">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>501</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Mono</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Set Item Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="LabelDescription">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>501</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Mono</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Set Item Description</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="SetItemDescription">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>511</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Mono</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid black;
border-radius: 6px;
padding-left: 3px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
   <property name="placeholderText">
    <string>Description [optional]</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="SetUnitSuffix">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>511</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Mono</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid black;
border-radius: 6px;
padding-left: 3px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
   <property name="placeholderText">
    <string>Kg,ft,lt,m3</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="SetItemName">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>511</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Mono</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid black;
border-radius: 6px;
padding-left: 3px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
   <property name="placeholderText">
    <string>Please set item's name</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CancelButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>336</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	color:white;
	border: 2px solid black;
	border-radius: 8px;
	background-color: rgba(220, 0, 0, 180);
}
QPushButton:hover {
	background-color: rgba(180, 0, 0, 200);
}
QPushButton:pressed {
	background-color: rgba(140, 0, 0, 200);
}</string>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
   <property name="iconSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="LabelUnit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>501</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Mono</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Set Unit Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="LabelUnitCost">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>501</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Mono</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Set Unit Cost</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>-1</y>
     <width>551</width>
     <height>411</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid white;
background-color: rgb(235, 235, 235);
border-radius: 8px;</string>
   </property>
   <widget class="QLabel" name="AlertText">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>380</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Pixeloid Sans</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:red;
border: 0px solid white;</string>
    </property>
    <property name="text">
     <string>Empty Data!</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QDoubleSpinBox" name="SetUnitCost">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>511</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QDoubleSpinBox {
	border: 2px solid black;
	border-radius: 8px;
	font-family: Noto Mono;
	font-size: 18px;
}</string>
   </property>
   <property name="suffix">
    <string> $</string>
   </property>
   <property name="maximum">
    <double>9999999999.000000000000000</double>
   </property>
  </widget>
  <zorder>widget</zorder>
  <zorder>AcceptButton</zorder>
  <zorder>LabelStoreItemName</zorder>
  <zorder>LabelDescription</zorder>
  <zorder>SetItemDescription</zorder>
  <zorder>SetUnitSuffix</zorder>
  <zorder>SetItemName</zorder>
  <zorder>CancelButton</zorder>
  <zorder>LabelUnit</zorder>
  <zorder>LabelUnitCost</zorder>
  <zorder>SetUnitCost</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
